backend/api_handler.py

```diff
@@ -42,7 +45,6 @@ def adapted(obj, info, **kwargs):
                 username=info.context['username'],
                 groups=info.context['groups'],
                 schema=info.context['schema'],
-                cdkproxyurl=info.context['cdkproxyurl'],
```

Comment:
[Clarification] - Like the es (ElasticSearch/OpenSearch) engine parameter, cdkproxyurl is now part of the context

Reply:
It's a part of a config. Config is something global and cdkproxyurl is a good match for a property. So I decided to put it there